src/main.cpp

```diff
@@ -131,7 +131,7 @@ typedef BOOL  (WINAPI *pMiniDumpWriteDump)(
 
 pMiniDumpWriteDump g_WriteMiniDump;  // The function in dbghlp DLL that creates dump files
 
-char g_DumpFileName[MAX_PATH];  // Filename of the dump file; hes to be created before the dump handler kicks in
+wchar_t g_DumpFileName[MAX_PATH];  // Filename of the dump file; hes to be created before the dump handler kicks in
```

Comment:
A question was posed [here](http://stackoverflow.com/questions/234365/is-tchar-still-relevant). The outcome is generally that unless you want to be backwards compatible, it's best to stick with Wide and be explicit about it.